Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -289,6 +289,28 @@ var ListView = React.createClass({
     }
   },
 
+  /**
+   * If this is a vertical ListView scrolls to the bottom.
+   * If this is a horizontal ListView scrolls to the right.
+   *
+   * Use `scrollToEnd()` for immediate scrolling,
+   * `scrollToEnd({animated: true})` for animated scrolling.
+   *
+   * See `ScrollView#scrollToEnd`.
+   */
+  scrollToEnd: function(options?: { animated?: boolean }) {
```

Comment:
Maybe we should expose a `getScrollComponent` method so we don't need to forward all of these methods.